central/usage/datastore/datastore_impl.go

```diff
@@ -17,18 +20,39 @@ var (
 type dataStoreImpl struct {
 	clustore clustore
 	cache    cache.Cache
+	store    postgres.Store
 }
 
 var _ DataStore = (*dataStoreImpl)(nil)
 
 // Get returns the object, if it exists from the store.
-func (ds *dataStoreImpl) Get(_ context.Context, _ *types.Timestamp, _ *types.Timestamp) ([]*storage.Usage, error) {
-	return nil, errors.New("not implemented")
+func (ds *dataStoreImpl) Get(ctx context.Context, from *types.Timestamp, to *types.Timestamp) ([]*storage.Usage, error) {
```

Comment:
It will be great if you add tests for this even using mocked store to test all cases as there are more than I expected.